### ISSUE TITLE
Spaces before and after variable throws validity error

### ISSUE THREAD

**Issue by sarabarrerariano** [tonyganch/gonzales-pe#242]
Error message given -> Please check validity of the block starting from line #31

Example:
This doesn' work
`src: url($themepath_base + 'fonts/Interstate-ExtraLightItalic.eot');
`
This works (notice the deleted spaces between the variable)
`src: url($themepath_base+'fonts/Interstate-ExtraLightItalic.eot');`

**Comment by DanPurdy** [comment]
The parse error is already fixed in the dev branch and should be released shortly.

@tonyganch @bgriffith 

The actual parse tree looks incorrect to me though for the following example, duplicated declaration delimiters and the operator is missing from the uri content too.

```
.test {
  src: url($themepath_base + 'fonts/Interstate-ExtraLightItalic.eot');
}
```

```
{
  "type": "stylesheet",
  "content": [
    {
      "type": "ruleset",
      "content": [
        {
          "type": "selector",
          "content": [
            {
              "type": "class",
              "content": [
                {
                  "type": "ident",
                  "content": "test"
                }
              ]
            }
          ]
        },
        {
          "type": "space",
          "content": " "
        },
        {
          "type": "block",
          "content": [
            {
              "type": "space",
              "content": "\n\n  "
            },
            {
              "type": "declaration",
              "content": [
                {
                  "type": "property",
                  "content": [
                    {
                      "type": "ident",
                      "content": "src"
                    }
                  ]
                },
                {
                  "type": "propertyDelimiter",
                  "content": ":"
                },
                {
                  "type": "space",
                  "content": " "
                },
                {
                  "type": "value",
                  "content": [
                    {
                      "type": "uri",
                      "content": [
                        {
                          "type": "raw",
                          "content": "$themepath_base"
                        },
                        {
                          "type": "space",
                          "content": " "
                        }
                      ]
                    },
                    {
                      "type": "space",
                      "content": " "
                    },
                    {
                      "type": "string",
                      "content": "'fonts/Interstate-ExtraLightItalic.eot'"
                    }
                  ]
                }
              ]
            },
            {
              "type": "declarationDelimiter",
              "content": ";"
            },
            {
              "type": "declarationDelimiter",
              "content": ";"
            },
            {
              "type": "space",
              "content": "\n"
            }
          ]
        }
      ]
    }
  ]
}
```

**Comment by sarabarrerariano** [comment]
Have you got any idea when 'shortly' is? Are we talking about days, weeks, months?
Thanks for the fast response

**Comment by DanPurdy** [comment]
Afraid not, There's a few PR's waiting for some crucial fixes/ additions and then there will be a major version bump.
If you're coming from sass-lint or another tool then it means there will be a bit of updating required there too to accomodate these changes. I would honestly think it would be in the coming couple of weeks.

**Comment by sarabarrerariano** [comment]
I'm using sc5 styleguides and it uses gonzales.
That's why I'm stuck now at generating my styleguides. 
So that's a bit unfortunate...

Hope it will be fixed fast :)

**Comment by tonyganch** [comment]
@sarabarrerariano, we've got just one PR blocking the next release (#214), if the author doesn't respond in next couple of days, I'll fix the remaining comments myself.

/cc @varya
Since that will be a major release (v4.0), sc5 team will need to update their `package.json` file. I hope there will be no breaking changes for them requiring code update.

@DanPurdy, I'll take a look at parsing tree.

**Comment by tonyganch** [comment]
@DanPurdy, both issues (double declaration delimiters and no operator) will be fixed with Ben's #217

**Comment by sarabarrerariano** [comment]
Any idea when the PR will be accepted and merged? (No pressure :) )
And hopefully there will be no breaking changes when using sc5

**Comment by DanPurdy** [comment]
The next breaking of gonzales is going to be v4 so I would expect breaking changes.

**Comment by tonyganch** [comment]
Hey @sarabarrerariano, the release is planned for tomorrow, 17.11

**Comment by tonyganch** [comment]
#217 is merged and published as v4.0.3, closing the issue.
